lib/ar2dto/dto.rb

```diff
@@ -6,17 +6,22 @@ def self.[](original_model)
       Class.new(self) do
         include ::AR2DTO::ActiveModel if original_model.ar2dto.active_model_compliance
         define_singleton_method(:original_model) { original_model }
+        attr_reader(*original_model.attribute_names)
       end
     end
 
-    def initialize(attributes = {})
-      attributes.each { |key, value| define_singleton_method(key) { value } }
-      super()
-    end
-
-    def as_json(options = nil)
```

Comment:
Is this method no longer necessary?

Reply:
right, `as_json` is defined by Rails on objects by building a hash with the instance vars that's exactly what we want